protocols/nonexistent.py

```
@@ -2,14 +2,23 @@
 # Previous failure rate: 100.00%
 # Mutation timestamp: {"timestamp": "2025-06-17T22:57:30.605986"}
 
+
 def task():
```

Ruff (0.11.9)

Missing return type annotation for public function `task`

(ANN201)